src/browser.ts

```diff
@@ -0,0 +1,148 @@
+import puppeteer, { ElementHandle } from 'puppeteer';
+import { Page, Browser as PuppeteerBrowser } from 'puppeteer';
+import { sleep } from './util/sleep';
+import { LikeOptions } from './luckybot';
+
+const host = 'https://www.instagram.com';
+
+export interface BrowserOptions {
+  debug?: boolean;
+}
+
+export class Browser {
+  browser?: PuppeteerBrowser;
+  page?: Page;
+  isLoggedIn: boolean = false;
+  options: BrowserOptions;
+
+  constructor(options?: BrowserOptions) {
+    this.options = options || {debug: false};
+  }
+
+  async createPage() {
+    const browser = await puppeteer.launch({ headless: !this.options.debug });
+    const page = await browser.newPage();
+    const userAgent = await browser.userAgent();
+    await page.setViewport({width: 1024, height: 768});
+    await page.setUserAgent(userAgent.replace('Headless', ''));
```

Comment:
Any idea about user agent?

Reply:
You mean, to set an user agent that seems real?